src/CONST.ts

```diff
@@ -1774,6 +1774,7 @@ const CONST = {
     },
     CENTRAL_PANE_ANIMATION_HEIGHT: 200,
     LHN_SKELETON_VIEW_ITEM_HEIGHT: 64,
+    LHN_AMOUNT_ITEMS_IN_VIEW_PORT: 20,
```

Comment:
```suggestion
    LHN_VIEWPORT_ITEM_COUNT: 20,
```
- `viewport` instead of `view port` as a correct english term
- `count` instead of `amount`, as a`amount` is for uncountable nouns (gets me every time too)